### PR TITLE
ZIOS-11423: Add script to conditionally embed internal settings

### PR DESCRIPTION
**Pull request by alexaubry**
### Issues

The Settings bundle is displayed in public builds.

### Causes

The new build scripts do not check whether the resource is internal before copying it.

### Solutions

Add a build variable to determine whether the build is internal or not, and whether we should copy the resource or not.

```bash
set -e

if [ $IS_INTERNAL_BUILD == '1' ]; then
    for(( i=0; i < $SCRIPT_INPUT_FILE_COUNT; i++ )); do
        INPUT_FILE="SCRIPT_INPUT_FILE_$i"
        cp -Ra "${!INPUT_FILE}" "$BUILT_PRODUCTS_DIR/$PRODUCT_NAME.app/"
    done
else
    for(( i=0; i < $SCRIPT_OUTPUT_FILE_COUNT; i++ )); do
        OUTPUT_FILE="SCRIPT_INPUT_FILE_$i"
        rm -rf "${!OUTPUT_FILE}"
    done
fi
```

### CODE REVIEW

**Comment by typfel** [comment]
You deleted the `Settings.bundle`?

**Comment by alexaubry** [comment]
@typfel No I moved it to a different folder (Resources/Internal)

**Comment by typfel** [comment]
@alexaubry then I think you forgot to check it in

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -6788,6 +6795,26 @@
...
+			);
+			name = "Embed Internal Resources";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+				"$(BUILT_PRODUCTS_DIR)/$(PRODUCT_NAME).app/Settings.bundle/",
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "set -e\n\nif [ $IS_INTERNAL_BUILD == '1' ]; then\n    for(( i=0; i < $SCRIPT_INPUT_FILE_COUNT; i++ )); do\n        INPUT_FILE=\"SCRIPT_INPUT_FILE_$i\"\n        cp -Ra \"${!INPUT_FILE}\" \"$BUILT_PRODUCTS_DIR/$PRODUCT_NAME.app/\"\n    done\nelse\n    for(( i=0; i < $SCRIPT_OUTPUT_FILE_COUNT; i++ )); do\n        OUTPUT_FILE=\"SCRIPT_INPUT_FILE_$i\"\n        rm -rf \"${!OUTPUT_FILE}\"\n    done\nfi\n";
```
Comment (typfel) [inline Wire-iOS.xcodeproj/project.pbxproj]: How about moving the script into an external file in the `Scripts/` folder? This is not very readable in PRs.

Comment (alexaubry) [reply Wire-iOS.xcodeproj/project.pbxproj]: Good idea.

**Comment by alexaubry** [comment]
@typfel Updated with your feedback.

**Pull request merged**